### PR TITLE
Declare Dart 2.0 compatability

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

**Comment by mit-mit** [comment]
cc @sortie can you review, please? (just leave a LGTM here in comments)

**Comment by mit-mit** [comment]
cc @kevmoo to confirm that this is the type of change we want to make in general. We tested that this works fine with a 2.0 stable build (except that it still fails as the deps of this package are not updated)

**Comment by mit-mit** [comment]
Moving discussion to https://github.com/dart-lang/protobuf/pulls as this was merged in order for us to test

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.6.0 - 2018-07-12
```
Comment (kevmoo) [inline CHANGELOG.md]: Avoid breaking change releases for this type of update. Will require folks to update their pubspecs, which is not needed!

**Pull request merged**